wpe/src/main/java/com/wpe/wpeview/WPEView.java

```diff
@@ -223,6 +228,49 @@ public void run()
         });
     }
 
+    public void enterFullScreen()
+    {
+        runOnMainThread(new Runnable()
+        {
+            @Override
+            public void run()
+            {
+                removeView(m_wpeSurfaceView);
+
+                m_customView = new FrameLayout(m_context);
+                m_customView.addView(m_wpeSurfaceView);
+                m_customView.setFocusable(true);
+                m_customView.setFocusableInTouchMode(true);
+
+                WebChromeClient.CustomViewCallback cb = () -> {
+                    if (m_customView != null) {
+                        Browser.getInstance().requestExitFullscreenMode(WPEView.this);
+                    }
```

Comment:
{} not useful

Reply:
follows the convention that was already used elsewhere in the file for similar code. We can reformat whole file later with consistent convention